NitroxInstallerActions/InstallPatchAction.cs

```diff
@@ -16,7 +16,7 @@ public static ActionResult InstallPatch(Session session)
             try
             {
                 string managedDirectory = session.CustomActionData["MANAGEDDIR"];
-                                
+                string gameDir = Path.GetDirectoryName(Path.Combine(Path.GetFullPath(managedDirectory), "..\\..\\Subnautica.exe"));
```

Comment:
Best way is to use the "linux path fix" like in #595 so it will be the same like the other code